ChatQnA/docker_compose/intel/cpu/xeon/compose.yaml

```diff
@@ -124,7 +124,7 @@ services:
       - RERANK_SERVER_HOST_IP=tei-reranking-service
       - RERANK_SERVER_PORT=${RERANK_SERVER_PORT:-80}
       - LLM_SERVER_HOST_IP=vllm-service
-      - LLM_SERVER_PORT=${LLM_SERVER_PORT:-80}
```

Comment:
why can we keep the LLM_SERVER_PORT for customizations?  for FAQ integration, it could define another variable for FAQ env variable like LLM_HOST_PORT_FAQ

Reply:
customizing LLM_SERVER_PORT  should be defined in line 88. Line 127 is used for internal port querying aligned with Line 126, it must be 80.